wpilibj/src/test/java/edu/wpi/first/wpilibj/event/BooleanEventTest.java

```diff
@@ -30,43 +30,44 @@ void testBinaryCompositions() {
     assertEquals(1, orCounter.get());
   }
 
+  /**
+   * When a BooleanEvent is constructed, an action is bound to the event loop to update an internal
+   * state variable. This state variable is checked during loop polls to determine whether or not to
+   * execute an action. If a condition is changed during the loop poll but before the state variable
+   * gets updated, the changed condition is used for the state variable.
+   */
   @Test
-  void testBinaryCompositionLoopSemantics() {
```

Comment:
Is a test being removed here? If so, why?

Reply:
It used to test that two events composed with different event loops would use the event loop from the first event. That's not applicable anymore because you now need to use the same event loop.

Reply:
Using different loops should still have a behavior that is explicitly tested -- especially if we already have a test for it.